Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py

```diff
@@ -19,6 +19,7 @@
 urllib3.disable_warnings()
 
 """ ADVANCED GLOBAL PARAMETERS """
+REQUEST_TIMEOUT = 20                # number of seconds to wait for a request result
```

Comment:
20 seconds is enough?

Reply:
should be, but I increased it to 30 just in case. It's customizable through the advanced params in any case.